src/Emulator/Peripherals/Peripherals/Timers/NRF52840_RTC.cs

```diff
@@ -89,6 +89,19 @@ private void DefineRegisters()
                     })
                     .WithReservedBits(1, 31)
                 },
+                {(long)Register.Clear, new DoubleWordRegister(this)
+                    .WithFlag(0, FieldMode.Write, name: "TASKS_CLR", writeCallback: (_, value) =>
```

Comment:
I know, however I though I keep it consistent with the file NRF52840_GPIOTasksEvents.cs where TASKS_CLR is used. let me know what you prefer.

Reply:
renamed

Reply:
This is because for GPIOTE the field is called "TASKS_CLR". Consistency? who cares? ;)
We try to keep register names in our models consistent with the documentation to make it easier to ctrl-f it later.